docs/bir-spec/src/test/java/org/ballerinalang/birspec/BIRTestUtils.java

```diff
@@ -155,7 +154,7 @@ private static void assertFunctions(BIRNode.BIRPackage expectedBIR, Bir.Module b
             assertBasicBlocks(actualFunctionBody.functionBasicBlocksInfo(), expectedFunction.basicBlocks,
                     constantPoolEntries);
 
-            assertScopes(actualFunction.scopeEntries(), expectedFunction);
+//            assertScopes(actualFunction.scopeEntries(), expectedFunction);
```

Comment:
Why is this commented?

Reply:
Added cases for terminators as well. Enabled this test.